.travis.yml

```diff
@@ -3,27 +3,22 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - '8'
+  - "8"
 
 sudo: false
 dist: trusty
 
 addons:
   chrome: stable
 
-cache: yarn
+cache:
+  yarn: true
 
 env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
-branches:
-  only:
-    - master
-    # npm version tags
-    - /^v\d+\.\d+\.\d+/
-
```

Comment:
Is there a reason why we're removing this?

Reply:
Ups. This was only required for testing my feature branch. Restored in 21a229c6c996bbac16a3a1eb247953afb251188b.